plugins/ckan_operators/datastore_operator.py

```diff
@@ -249,3 +301,88 @@ def execute(self, context):
         logging.info(f"Result: {result}")
 
         return result
+
+
+
+class InsertDatastoreResourceRecordsFromJSONOperator(BaseOperator):
```

Comment:
This is like the `InsertDatastoreResourceRecordsOperator` except tailored to read a JSON file.